firebase-vscode/webviews/globals/app.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactNode, StrictMode } from "react";
+import { ExtensionStateProvider } from "./extension-state";
+
+export function App({ children }: { children: ReactNode }): JSX.Element {
```

Comment:
Comment on the purpose of this (to wrap every webview in a common template, which to start off with is just StrictMode)

Reply:
Ack, that's right, and added a comment.